test/simulator-e2e-specs.js

```diff
@@ -294,6 +294,13 @@ if (!process.env.TRAVIS) {
         device: 'iPhone 5s'
       }
     ];
+  } else if (process.env.DEVICE === '10' || process.env.DEVICE === '10.0') {
+    deviceTypes = [
+      {
+        version: '10.0',
+        device: 'iPhone 5s'
```

Comment:
very minor, but do you think the default device for ios 10 should be something more recent like a 6s?